### PR TITLE
DBZ-4389: Update Vitess docs to add keepalive pings and transaction metadata

### PR DESCRIPTION
**Pull request by shichao-an**
- Add Transaction metadata section
- Add config properties: `vitess.keepalive.interval.ms`, `provide.transaction.metadata`, `transaction.topic`

This is related to https://issues.redhat.com/browse/DBZ-4389 and also fixes https://issues.redhat.com/browse/DBZ-4361

cc @sonne5 @gunnarmorling

### CODE REVIEW

File: documentation/modules/ROOT/pages/connectors/vitess.adoc
Comment (gunnarmorling) [inline documentation/modules/ROOT/pages/connectors/vitess.adoc:1263]: Could you use the xref syntax (if there's no example for that in this doc, you can find it in others, e.g. the PG one) instead of link-prefix? The latter is a left-over from a time where we couldn't use xref for some issue when rendering downstream product docs; but this has been resolved and link-prefix should be phased out in favour of the more concise xref. I know it's not handled consistently like that yet, but let's avoid adding more link-prefix usages in new/changed docs.

**Review by gunnarmorling — APPROVED** [review]
LGTM, one small comment inline.

Btw. if you want to preview the rendered doc locally, you can do so as described [here](https://github.com/debezium/debezium.github.io#22-using-the-container-image---generate-debezium-docs-from-local-repo).

**Comment by sonne5** [comment]
LGTM, pending on Gunnar's comment

**Review by sonne5 — APPROVED** [review]

File: documentation/modules/ROOT/pages/connectors/vitess.adoc
Comment (shichao-an) [reply documentation/modules/ROOT/pages/connectors/vitess.adoc:1263]: Thanks for calling this out. I'm not really familiar of the xref syntax and not sure what part should be changed.
I checked https://github.com/debezium/debezium/blame/main/documentation/modules/ROOT/pages/connectors/postgresql.adoc#L3130  and it seems it's also using link-prefix for the counterpart config section on transaction metadata: `{link-prefix}:{link-postgresql-connector}#postgresql-transaction-metadata[Transaction metadata]`.

It would be great if there's a how-to guide or canonical example to follow. We might as well want to have a refactoring pass to fix docs consistently

**Comment by shichao-an** [comment]
> LGTM, one small comment inline.
> 
> Btw. if you want to preview the rendered doc locally, you can do so as described [here](https://github.com/debezium/debezium.github.io#22-using-the-container-image---generate-debezium-docs-from-local-repo).

Thanks for the tips, this is useful. I was using an Intellij plugin to view it instantly.

**Comment by shichao-an** [comment]
Looks like i don't have access to merge this PR. 😂  I may need someone to help merge this

File: documentation/modules/ROOT/pages/connectors/vitess.adoc
Comment (gunnarmorling) [reply documentation/modules/ROOT/pages/connectors/vitess.adoc:1263]: You can find an example here: https://github.com/debezium/debezium/blame/main/documentation/modules/ROOT/pages/connectors/postgresql.adoc#L343. I.e. it's `xref:reference-id[]`, or `xref:reference-id[some text]`. More details in the [Antora docs](https://docs.antora.org/antora/3.0/page/xref/).

Comment (gunnarmorling) [reply documentation/modules/ROOT/pages/connectors/vitess.adoc:1263]: > We might as well want to have a refactoring pass to fix docs consistently

Yeah, if you wanted to change this completely for the Vitess docs, that'd be great.

**Comment by gunnarmorling** [comment]
Ok, so this LGTM. I'll merge and just push another commit for changing the link to xref syntax as discussed. Thx!

> Looks like i don't have access to merge this PR. 

Yes, commit access to this repo is a bit more restricted, for now I'll just take care of merging any PRs here. I don't think this should impact you too much for the Vitess-related work; if it does, we can reconsider this down the road. I hope that's ok for the time being?

**Comment by shichao-an** [comment]
> Ok, so this LGTM. I'll merge and just push another commit for changing the link to xref syntax as discussed. Thx!

Thank you Gunnar for taking care of that as part of this PR!

> Yes, commit access to this repo is a bit more restricted, for now I'll just take care of merging any PRs here. I don't think this should impact you too much for the Vitess-related work; if it does, we can reconsider this down the road. I hope that's ok for the time being?

Thanks! Sounds good to me.

**Pull request merged**
